docs/alerting-recording.md

```diff
@@ -0,0 +1,59 @@
+# Alerting & Recording Rules
+
+### Alerting Rules
+
+The alerting rules are used to trigger alerts based on the provided alerting rules an alert should be fired to an external service like slack. In promscale the alerting rules are evaluated in prometheus. i.e define alerting rules in prometheus config
+
+```
+groups:
+  - name: <alert-group-name>
+    rules:
+    - alert: <alert-name>
+      expr: <promql_expression>
+      for: <time-interval for how long this to happen to happen to fire an alert>
+      labels:
+        <key>:<value>
+      annotations:
+        summary: <text>
+        description: <description on alert>
+```
+
+Load the above defined alert rule file to your prometheus config for evaluation. i.e 
+
+```
+rule_files:
+  - "<alert-rules-file>"
+```
+
+Configuring the alert manager in prometheus config
+
+```
+alerting:
+  alertmanagers:
+  - static_configs:
+    - targets:
+      - localhost:9093
+```
+
+
+### Recording Rules
+
+The recording rules are for pre-calculating frequently used or computationally expensive queries. The results of those rules are saved into their own time series. In promscale the rules rules are evaluated in prometheus. i.e define recording rules in prometheus config
```

Comment:
please mention that these newly formed time-series follow the name of the record (please confirm once)